scripts/west_commands/build.py

```diff
@@ -64,6 +65,16 @@ class AlwaysIfMissing(argparse.Action):
     def __call__(self, parser, namespace, values, option_string=None):
         setattr(namespace, self.dest, values or 'always')
 
+class CommaDelimitedList(argparse.Action):
+    def __call__(self, parser, namespace, values, option_string=None):
+        result = []
+        for value in values:
+            # Split the value by commas, then strip whitespace from each item
+            items = [item.strip() for item in value.split(',')]
+            # Add non-empty items to the result list
+            result.extend(item for item in items if item)
+        setattr(namespace, self.dest, result)
```

Comment:
Not a big fan of this, we should stick to the default argument parser handling of appending (action `append`). This looks like a whole can of worms to be opened.

Reply:
would look like "--extra-conf foo.conf,bar.conf" which may be confusing to use. i can move this to a separate PR

Reply:
I think `--extra-conf foo.conf --extra-conf bar.conf` is not too big of a hurdle, and is more explicit.